lib/stat_tracker.rb

```diff
@@ -1,36 +1,26 @@
-require_relative "./game_collection"
-require_relative "./team_collection"
-require_relative "./gt_collection"
+require_relative './game'
+require_relative './team'
+require_relative './game_team'
 
 class StatTracker
-  attr_reader :games_path, :teams_path, :game_teams_path, :game_collection, :team_collection
+  attr_reader :games_path, :teams_path, :game_teams_path, :games, :teams, :game_teams
 
   def initialize(stat_tracker_params)
     @games_path = stat_tracker_params[:games]
     @teams_path = stat_tracker_params[:teams]
     @game_teams_path = stat_tracker_params[:game_teams]
-    @game_collection = GameCollection.new(@games_path)
-    @team_collection = TeamCollection.new(@teams_path)
-    @gt_collection = GameTeamCollection.new(@game_teams_path)
+
+    @games = Game.all(@games_path)
+    @teams = Team.all(@teams_path)
+    @game_teams = GameTeam.all(@game_teams_path)
```

Comment:
This looks good for each class, but the paths need to be stored as instance variables.

Reply:
The paths DON'T need to be stored****